schemas/context/experienceevent-enduserids.schema.json

```diff
@@ -7,19 +7,19 @@
   ],
   "$id": "https://ns.adobe.com/xdm/context/experienceevent-enduserids",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "ExperienceEvent endUserIDs",
+  "title": "ExperienceEvent EndUserIDs",
```

Comment:
Check - I think endUserIDs might be correct.

Reply:
Fixed!